packages/fluentui/react-northstar/src/themes/teams/components/Avatar/avatarLabelStyles.ts

```diff
@@ -0,0 +1,42 @@
+import { ComponentSlotStylesPrepared, ICSSInJSStyle } from '@fluentui/styles';
+
+import { AvatarVariables } from './avatarVariables';
+import { pxToRem, SizeValue } from '../../../../utils';
+import { AvatarLabelStylesProps } from '../../../../components/Avatar/AvatarLabel';
+
+const sizeToPxValue: Record<SizeValue, number> = {
+  smallest: 20,
+  smaller: 24,
+  small: 28,
+  medium: 32,
+  large: 44,
+  larger: 64,
+  largest: 96,
+};
+
+export const avatarLabelStyles: ComponentSlotStylesPrepared<AvatarLabelStylesProps, AvatarVariables> = {
+  root: ({ props: p, variables: v }): ICSSInJSStyle => {
+    const sizeInRem = pxToRem(sizeToPxValue[p.size]);
+    return {
+      alignItems: 'center',
+      overflow: 'hidden',
+      color: v.labelBackground,
+      backgroundColor: v.labelForeground,
```

Comment:
```suggestion
      color: v.labelColor,
      backgroundColor: v.labelBackground,
```

Let's bee consistent in naming 🐱

Reply:
Done